airflow/www/views.py

```diff
@@ -1024,12 +1024,12 @@ def blocked(self, session=None):
             )
             if 'all_dags' not in filter_dag_ids:
                 dags = dags.filter(DR.dag_id.in_(filter_dag_ids))
-            dags = dags.all()
+            else:
```

Comment:
I am not sure if this is correct. [all()](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.all) collects the dags - [filter()](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.filter) does only filter the dags but not collect them.

Reply:
Yeah, that's true. Thanks!